prep-html-relative-pathing/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div><a href="/Users/Jimbodini/lfz/prep-0121-code-solutions/prep-html-relative-pathing/index.html">Click here to go to the home page</a></div>
```

Comment:
Oops! This is an absolute path, not a relative path. Please revise this to use a relative path as described in the documentation for the exercise 👍